Lab_2.py

```diff
@@ -0,0 +1,80 @@
+import string
+import logging
+import sys
+from collections import Counter
+
+if __name__ == '__main__':
+    if len(sys.argv) < 2:
+        print('Please provide logging file name as argument')

```

Comment:
This is another convention thing.  Typically the log names follow the script/app name.  For example, a tokenizer.py file would be expected to generate a tokenizer.log.  In fact, the color syntanx highlighting triggers on 'log' in most shells.  So ya, again, not wrong, but typically you ask for flags for the operation of the functions and the log names are defaulted.